iOS/ViewControllers/Courses/CourseViewController.swift

```diff
@@ -73,6 +74,24 @@ class CourseViewController: UIViewController {
         set {}
     }
 
+    private lazy var actionMenuButton: UIBarButtonItem = {
+        let button = UIBarButtonItem(image: R.image.dots(), style: .plain, target: self, action: #selector(showActionMenu(_:)))
+        button.isEnabled = true
+        return button
+    }()
+
+    private var shareCourseAction: UIAlertAction {
+        return UIAlertAction(title: NSLocalizedString("courseIteam.share", comment: "Title for course item share action"), style: .default) { [weak self] _ in
```

Comment:
Typo ;) 

Please use a better key for the localizable string. Something like `course.action-menu.share`